doc/users/whats_new/style_changes.rst

```diff
@@ -0,0 +1,98 @@
+Changes to the default style
+----------------------------
+
+The most important changes in matplotlib 2.0 are the changes to the
+default style.
+
+While it is impossible to select the best default for all cases, these
+are designed to work well in the most common cases.
+
+These changes include:
+
+Colors
+``````
+
+- The default figure background color has changed from grey to white.
+
+- The default cycle of colors to draw lines, markers and other content
+  has been changed.  It is based on the `Vega category10 palette
+  <https://github.com/vega/vega/wiki/Scales#scale-range-literals>`__.
+
+- The default color map used for images and pcolor meshes, etc., has
+  changed from ``jet`` to ``viridis``.
+
+- For markers, scatter plots, bar charts and pie charts, there is no
+  longer a black outline around filled markers by default.
+
+- Grid lines are light grey solid 1pt lines.  They are no longer dashed by
+  default.
+
+- The ''Blues'' colormap has been adjusted to be perceptually uniform.
+  The old ``blues`` colormap is available under the name
+  ``legacy_Blues``.
+
+Plot layout
+```````````
+
+- The default dpi used for on-screen is now 100, which is the same as
+  the old default for saving files.  Due to this, the on-screen
+  display is now more what-you-see-is-what-you-get.
+
+- The number of ticks on an axis is now automatically determined based
+  on the length of the axis.
+
+- The limits are scaled to exactly the dimensions of the data, plus 5%
+  padding.  The old behavior was to scale to the nearest "round"
+  numbers.  To use the old behavior, set the ``rcParam``
+  ``axes.autolimit_mode`` to ``round_numbers``.
+
+- Ticks now point outward by default.  To have ticks pointing inward,
+  use the ``rcParams`` ``xtick.direction`` and ``ytick.direction``.
+
+- By default, caps on the ends of errorbars are not present.
+
+Images
+``````
+
+- The default mode for image interpolation is now ``nearest``.
+
+- The default shading mode is now ``overlay``.  Formerly, it was
```

Comment:
Maybe this should reference the function where that applies? It doesn't appear to be controlled by an `rcParam`.